remotespark/utils/constants.py

```diff
@@ -60,15 +60,16 @@
 NOT_STARTED_SESSION_STATUS = "not_started"
 STARTING_SESSION_STATUS = "starting"
 BUSY_SESSION_STATUS = "busy"
+SUCCESS_SESSION_STATUS = "success"
```

Comment:
I've spoken to @tc0312 about this and he seemed to indicate that it is a bug if interactive Livy sessions over go to success state